docs/slack-apps/slack-packages.md

```diff
@@ -0,0 +1,65 @@
+---
+id: slack-packages
+title: Slack Packages
+sidebar_label: Slack Packages
+---
+
+We've built in a variety of functions to make it easier to build Dark apps with Slack. Use these functions from autocomplete (dark/slack/Slack::), the same way you would any other function.
+
+![Slack](assets/packages/slackpackage.png)
+
+### getAllConversations
+
+Takes your Slack token as a String and returns a list of Dicts containing both your public and private Slack channel names and ids.
+
+![Slack](assets/packages/getAllConversations.png)
+
+### getConversations
+
+Takes your Slack token as a String and returns a list of Dicts containing your public Slack channel names and ids.
+
+![Slack](assets/packages/getConversations.png)
+
+### getConversationsWithDetails
+
+Takes your Slack token as a string and returns a list of Dicts containing the channel name, id, created at date (in Epoch seconds), creator, count of members, purpose and current topic. Does not include archived channels.
+
+![Slack](assets/packages/getConversationsWithDetails.png)
+
+### getNewConversations
+
+Takes your Slack token as a String and checkFromSeconds as an int, and returns a list of conversations (as Dicts) where the created at date is greater than the current time minus the checkFromSeconds time. Each entry contains the channel name, id, created at date (in Epoch seconds), creator, count of members, purpose and current tppic.
+
+![Slack](assets/packages/getNewConversations.png)
+
+### getPrivateConversations
+
+Takes your Slack token as a String and returns a list of Dicts containing your private Slack channel names and ids.
+
+![Slack](assets/packages/getPrivateConversations.png)
+
+### oauth
+
+Takes your Slack clientID (String), clientSecret (String), token datastore (Any - but should be the name of your datastore), and oauth code (String) and returns a Result.
```

Comment:
This doesn't describe what the function does. It's also a fairly ambiguous name, so hard to figure it out without reading through the code.

Reply:
All records that we return should use camelCase, not underscores. Here they have underscores.